src/components/ContentSection/defContent.ts

```diff
@@ -0,0 +1,18 @@
+
+const def2 = `<p>Платформа Доказові заохочує використовувати, публікувати і розповсюджувати оприлюднені на ній матеріали. Водночас використання матеріалів можливе за умов, викладених у цих правилах.</p><p>Матеріали, оприлюднені на платформі Доказові, створені для цієї платформи, якщо в самій публікації не вказано інше джерело (наприклад, що це допис експерта у соцмережах або відео організації).</p><p>Матеріали платформи дозволено використовувати:</p><p>в інших медіа –лише за умови посилання на платформу Доказові:</p><ul><li>для діджитал-медіа – гіперпосилання на dokazovi.info не нижче першого абзацу</li><li>для телебачення і радіо – згадка в синхроні сюжету/передачі</li><li>для особистих цілей: ви можете завантажити або скопіювати вміст сайту, інші компоненти та елементи, що відображаються на сайті</li><li>для освітнього процесу: як джерела для матеріалів занять, завдання тощо</li></ul><p>Під час використання матеріалів у будь-якому випадку не можна змінювати текст і візуальні матеріали, щоб запобігти тому, що сенс матеріалів може бути розтлумачено невірно, виділено з контексту або викривлено.</p><p>В будь-якому випадку під час використання матеріалів необхідно завжди вказувати його автора/авторів.</p><p>Використовувати матеріали платформи з метою отримання комерційної вигоди заборонено.</p><p><br></p><p>Переклади матеріалів видань ____, ____, _____ здійснено на умовах ексклюзивної угоди між ЮНІСЕФ в Україні та цими виданнями. УМОВИ ПЕРЕДРУКУ ЦИХ МАТЕРІАЛІВ.</p><p><em>Тут ще буде доповнення по тексту, поки за оцінками – до 1000 символів.</em></p>`;
+
+const def3 = `<p>Хочете стати автором платформи?</p><p>Бажаєте співпрацювати?</p><p>Маєте запитання?</p><p>Пишіть нам на contact@dokazovi.info</p>`;
```

Comment:
question: why not create a component and add this content there as JSX? 

Then, `dangerouslySetInnerHTML` could be avoided